docs/user-guide/jobs.md

```diff
@@ -206,6 +215,22 @@ If you do not specify `.spec.parallelism`, then it defaults to `.spec.completion
 Depending on the pattern you are using, you will either set `.spec.completions`
 to 1 or to the number of units of work (see [Job Patterns] for an explanation).
 
+### Defaulting Rules for Parallelism and Completions
+
+- `parallelism = nil`, `completions = nil` defaults to `parallelism = 1`, `completions = 1`
```

Comment:
I'm not sure if preserving this option won't confuse users over what's happening in the next line. I'm inclined to remove this defaulting.